src/pages/builder/index.tsx

```diff
@@ -29,22 +22,10 @@ export default function BuilderPage() {
   const cvDataFromRedux = useAppSelector((state) => state.cv);
   const dispatch = useAppDispatch();
 
-  const {
-    introSectionFormData: { name, country, job },
-    skillsSectionFormData: { skills, hobbies },
-    workSectionFormData: { employer, position, dateTime, remark },
-    educationSectionFormData: { institution, field, studyDateTime, studyRemark },
-    introResultFromAI,
-    setIntroResultFromAI,
-    workResultFromAI,
-    setWorkResultFromAI,
-    educationResultFromAI,
-    setEducationResultFromAI
-  } = useResumeFormContext();
   const [messagesHistory, setMessagesHistory] = useState<Record<AIResumeTypes, ConversationHistory> | null>({
-    profile: [{ role: 'assistant', content: introResultFromAI ?? '' }],
-    workExperience: [{ role: 'assistant', content: workResultFromAI ?? '' }],
-    education: [{ role: 'assistant', content: educationResultFromAI ?? '' }]
+    profile: [{ role: 'assistant', content: '' }],
+    workExperience: [{ role: 'assistant', content: '' }],
+    education: [{ role: 'assistant', content: '' }]
```

Comment:
I'm not sure but perhaps we miss here a bit, part that is related to skills and hobbies, if so probably we have to add it

Reply:
we have to add the logic for rewriting with additional context